pkgs/applications/version-management/git-mit/default.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+, openssl
+, pkg-config
+}:
+let
+  pname = "git-mit";
+  owner = "PurpleBooth";
+  version = "5.12.146";
+in
+rustPlatform.buildRustPackage {
+  inherit pname version;
+  src = fetchFromGitHub {
+    inherit owner;
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-VN1TbK9wi5Nt2K3yKx2lYSP30zSpwNETQ4OyHj8zxBg=";
+  };
+
+  cargoHash = "sha256-YtUuRLjmehG+5kUiCo4LK0PkKAckr28UahlrAjm9MYw=";
+
+  # attempts to run the program on .git in src which is not deterministic
+  # doCheck = false;
```

```suggestion
  doCheck = false;
```

Why to keep it commented?

---

For some reason, when I was testing the build, it would check git-cliff instead of git-mit, I fixed it and forgot to push my changes 😅